lib/src/widgets/sky_button.dart

```diff
@@ -23,6 +23,7 @@ class SkyButton extends StatelessWidget {
   final int secondMaxLines;
   final Color secondColor;
   final FontWeight secondFontWeight;
+  final EdgeInsets buttonPadding;
```

Comment:
why isn't this called just 'padding'?

Reply:
because I wrote, just live with this.

Reply:
then you can live with your decisions very far away from this company :)